### ISSUE TITLE
HACK with URL–fields do not work well with simplified chinese translation

### ISSUE THREAD

**Issue by joseph2002** [sehmaschine/django-grappelli#756]
```
text = text.replace(/^\w*: /, "");
text = text.replace(/<br>.*: /, "");
```
These two line of javascript in change_form.html does not work with simplified chinese translation.
first, \w does not match chinese characters,
second, the colon is translated to chinese colon, which is \uff1a

replace them with just one line ``text = text.replace(/.*[:|\uff1a] /, "");`` seems work

**Comment by sehmaschine** [comment]
can anyone confirm that the proposed solution works with different languages?

**Comment by joseph2002** [comment]
maybe you can give a block around this script for override?
so that we can tweak it for different language, or just disable it to use original url-field.

**Comment by sehmaschine** [comment]
the block might be a good idea – I'll take a look.

**Comment by sehmaschine** [comment]
fixed with 78877e1be88836b13c52a993d115e8ecc65600fe
